add_route.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#
+# add_route.sh
+# Script to setup persistant TEDAPI Routing for Powerwall Dashboard
+# Version: 1.0.0
+# By Scott Hibbard  - 2024-09-14
+#
+
+CRONTAB="/var/spool/cron/crontabs/root"
+PW_IP=""
+SCRIPT_NAME="TDAPI_routing"
```

Comment:
I suggest we use "TEDAPI_routing" to match the api name.

Reply:
??? You mean change the name of add_route.sh?  script that cron will run is already named "TEDAPI_routing".

Reply:
No, I think this was just a typo. This says "TDAPI_routing" but we should rename it "TEDAPI_routing" (it is missing the E).

Reply:
Oops, not enough coffee earlier. Thx.